README.md

```diff
@@ -0,0 +1,61 @@
+# community.hashi_vault Collection
+<!-- Add CI and code coverage badges here. Samples included below. -->
+[![CI](https://github.com/ansible-collections/community.hashi_vault/workflows/CI/badge.svg?event=push)](https://github.com/ansible-collections/community.hashi_vault/actions) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/community.hashi_vault)](https://codecov.io/gh/ansible-collections/community.hashi_vault)
+
+<!-- Describe the collection and why a user would want to use it. What does the collection do? -->
+
+## Tested with Ansible
+
+<!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
```

Comment:
the tested versions of ansible aren't listed in the `galaxy.xml` are they? Is this referencing something else?

Reply:
It's not mentioned anywhere. (meta/runtime.yml should say the collection supports 2.9.10+ or something like that, I forgot what we had in the collection template.)